get_deps.sh

```diff
@@ -291,6 +292,95 @@ EOF
     git clone -b task-bench https://github.com/elliottslaughter/x10.git "$X10_DIR"/x10
 fi
 
+if [[ $USE_HPX -eq 1 ]]; then
+    export HPX_DIR="$TASKBENCH_DEPS_DIR"/hpx
+    cat >>deps/env.sh <<EOF
+export HPX_DIR="\$TASKBENCH_DEPS_DIR"/hpx
+
+EOF
+
+    mkdir -p "$HPX_DIR"
+
+    cat >>"$HPX_DIR"/env.sh <<EOF
+
+export SOURCE_ROOT = "\$HPX_DIR"/src
+export INSTALL_ROOT = "\$HPX_DIR"/install
+
+export CMAKE_SRC_DIR =${SOURCE_ROOT}/cmake
+export CMAKE_BUILD_DIR =${INSTALL_ROOT}/cmake/build
+export CMAKE_INSTALL_DIR =${INSTALL_ROOT}/cmake
+
+export HWLOC_SRC_DIR =${SOURCE_ROOT}/hwloc
+export HWLOC_BUILD_DIR =${INSTALL_ROOT}/hwloc/build
+export HWLOC_INSTALL_DIR =${INSTALL_ROOT}/hwloc
+
+export BOOST_SRC_DIR =${SOURCE_ROOT}/boost
+export BOOST_INSTALL_DIR =${INSTALL_ROOT}/boost
+
+export JEMALLOC_SRC_DIR =${SOURCE_ROOT}/jemalloc
+export JEMALLOC_INSTALL_DIR =${INSTALL_ROOT}/jemalloc
+
+    DOWNLOAD_URL="https://github.com/Kitware/CMake/releases/download/v3.22/cmake-3.22.0.tar.gz"
+    if [[ ! -d ${CMAKE_SRC_DIR} ]]; then
+    (
```

Comment:
I don't see any variables being set here. Any reason for the subshell? (Same below.)